rocm_docs/rocm-release-manual.md

```diff
@@ -0,0 +1,65 @@
+How to synchonize TensorFlow ROCm port with upstream TensorFlow
+===============================================================
+
+This document articulates the step to create tensorflow rocm releases from upstream.
+
+
+Useful github remote repositories (For reference)
+-------------------------------------------------
+
+git locations of repositories used in the merge process are:
+
+- TensorFlow ROCm port:
+  - URL: <https://github.com/ROCm/tensorflow-upstream.git>
+  - branch: develop-upstream
+- TensorFlow upstream:
+  - URL: <https://github.com/tensorflow/tensorflow.git>
+  - branch: master
+
+# Step-by-step Merge Process
+--------------------------
+
+## Get TensorFlow ROCm port prior to the release branching on upstream
+
+If upstream branched out r2.19 on 2025-02-20, get verified weekly-sync prior to this date
```

Comment:
could you specific which weekly-sync branch you're using for r2.19 as the example? and the reason why? for example, because r2.19 is released at Feb 24 2025 and you can choose xxx


so other can reference it next time.